libraries/Wire/README.md

```diff
@@ -0,0 +1,19 @@
+# Wire
+
+The STM32 Wire library is fully compatible with the Arduino Wire specifications.
+
+## API note
+
+Two new method has been added to allow the use of large number of bytes to write
+or read.
+
+* **writeBuffer(uint8_t \* pdata, size_t quantity)**: pass pdata directly to the
+I2C layer withtout fill the internal Wire library buffer. This method must be
+called only once after _beginTransmission()_. _endTransmission()_ must be called
+immediately after. No other _write()_ method have to be called between
+_beginTransmission()_ and _endTransmission()_ when _writeBuffer()_ is used.
+
+* **requestFrom(uint8_t address, uint16_t quantity, uint8_t \*pdata)**: this
+function allows to read data and store them directly in pdata buffer.
+Do not call _available()_ and _read()_ after this function because data are
+directly stored in pdata buffer.
```

Comment:
Same here, if new methods are not easy to understand, I am not sure whether we should offer them